sklearn/utils/estimator_checks.py

```diff
@@ -211,6 +211,12 @@ def _boston_subset(n_samples=200):
         BOSTON = X, y
     return BOSTON
 
+def get_kwargs(estimator_class):
```

Comment:
I think if anything, this should be in ``set_fast_parameters`` and that function be renamed.
For the common tests it might be ok to catch the deprecation warnings when the decision function is used.